src/components/CustomDropdownComp.tsx

```diff
@@ -43,54 +47,82 @@ export default function CustomDropdownComp(props: CustomDropdownComponentProps):
     const [value, setValue] = useState<Option>();
 
     const createOption = (label: string, secondaryLabel: string, id: string, urlstring: string) => ({
-        label: <Label
-            DisplayName={label}
-            UrlString={urlstring}
-            ClassNamePrefix={props.classNamePrefix}
-            EnableAvatar={props.useAvatar}
-            SecondLabel={secondaryLabel}
-        />,
+        label: (
+            <Label
+                DisplayName={label}
+                UrlString={urlstring}
+                ClassNamePrefix={props.classNamePrefix}
+                EnableAvatar={props.useAvatar}
+                SecondLabel={secondaryLabel}
+            />
+        ),
         value: label,
+        // eslint-disable-next-line object-shorthand
         id: id,
         secondLabel: secondaryLabel,
         url: urlstring
     });
 
     useEffect(() => {
-        if (props.defaultValue.status === 'available') {
+        if (props.defaultValue.status === "available") {
```

Comment:
```suggestion
        if (props.defaultValue.status === ValueStatus.Available) {
```
That is also in [my PR](https://github.com/mendixlabs/CustomDropdown/pull/6/files#diff-15bf84ebcfbd6c361601d0f236a7e9a25bf33bbdd14e5cff8fc6048615c89dbdR40) which is still open ((((((